draft-ietf-httpbis-priority.md

```diff
@@ -150,64 +140,28 @@ in a revision of HTTP/2 {{HTTP2}}. However, in order to maintain wire
-to manipulate the server's stored prioritization state.
-
-HTTP/2 priority associated with an HTTP request is signalled as a value relative
-to those of other requests sharing the same HTTP/2 connection. Therefore, in
-order to prioritize requests, endpoints are compelled to have the knowledge of
-the underlying HTTP version and how the requests are coalesced. This has been
-a burden to HTTP endpoints that generate or forward requests in a
-version-agnostic manner.
-
-HTTP/2 priority signals are required to be delivered and processed in the order
-they are sent so that the receiver handling is deterministic. Porting HTTP/2
-priority signals to protocols that do not provide ordering guarantees presents
-challenges. For example, HTTP/3 {{HTTP3}} lacks global ordering across streams
-that would carry priority signals. Early attempts to port HTTP/2 priority
-signals to HTTP/3 required adding additional information to the signals, leading
-to more complicated processing. Problems found with this approach could not be
-resolved and definition of a HTTP/3 priority signalling feature was removed
-before publication.
-
-Considering the deployment problems and the design restrictions of RFC 7540
-stream priority, as well as the difficulties in adapting it to HTTP/3,
-continuing to base prioritization on this mechanism risks increasing the
-complexity of systems. Multiple experiments from independent research have shown
-that simpler schemes can reach at least equivalent performance characteristics
-compared to the more complex RFC 7540 setups seen in practice, at least for the
-web use case.
+Many RFC 7540 server implementations do not act on HTTP/2 priority
+signals. Prioritization can use information that servers have about resources or
+the order in which requests are generated. For example, a server, with knowledge
+of an HTML document structure, might want to prioritize the delivery of images
+that are critical to user experience above other images.
+
+RFC 7540 does not define a method that can be used by a server to provide a
+priority signal for intermediaries.
+
+RFC 7540 priority is expressed relative to other requests on the same connection.
+Many requests are generated without knowledge of how other requests might share a
+connection, which makes this difficult to use reliably, especially in protocols
+that do not have strong ordering guarantees, like HTTP/3 {{HTTP3}}.
+
+Multiple experiments from independent research have shown that simpler schemes
+can reach at least equivalent performance characteristics compared to the more
+complex RFC 7540 setups seen in practice, at least for the web use case.
 
 ## Disabling RFC 7540 Priorities {#disabling}
 
-The problems and insights set out above provided the motivation for deprecating
-RFC 7540 stream priority (see {{Section 5.3 of RFC7540}}).
+The problems and insights set out above provided the motivation for an
+alternative to RFC 7540 stream priority (see {{Section 5.3 of HTTP2}}).
```

Comment:
It seems odd to link to HTTP/2 when this section states disabling RFC 7540.

Reply:
My mistake.  I will revert this piece.

Reply:
```suggestion
alternative to RFC 7540 stream priority (see {{Section 5.3 of RFC7540}}).
```